sky/tools/create_macos_framework.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import subprocess
+import shutil
+import sys
+import os
+
+from create_xcframework import create_xcframework
+
+DSYMUTIL = os.path.join(os.path.dirname(__file__), '..', '..', '..',
+                        'buildtools', 'mac-x64', 'clang', 'bin', 'dsymutil')
+
+def main():
+  parser = argparse.ArgumentParser(description='Creates FlutterMacOS.framework for macOS')
+
+  parser.add_argument('--dst', type=str, required=True)
+  parser.add_argument('--arm64-out-dir', type=str, required=True)
+  parser.add_argument('--x64-out-dir', type=str, required=True)
+  parser.add_argument('--strip', action="store_true", default=False)
+  parser.add_argument('--dsym', action="store_true", default=False)
+
+  args = parser.parse_args()
+
+  fat_framework = os.path.join(args.dst, 'FlutterMacOS.framework')
+  arm64_framework = os.path.join(args.arm64_out_dir, 'FlutterMacOS.framework')
+  x64_framework = os.path.join(args.x64_out_dir, 'FlutterMacOS.framework')
+
+  arm64_dylib = os.path.join(arm64_framework, 'FlutterMacOS')
+  x64_dylib = os.path.join(x64_framework, 'FlutterMacOS')
```

Comment:
Could point to `Versions/A/FlutterMacOS` instead, but I guess `lipo` is following the symlink if script is working.

Reply:
I think you're right, but keeping as-is. We can adjust if we find that lipo isn't chasing symlinks.